src/components/Navigation.tsx

```diff
@@ -120,6 +45,96 @@ export const Navigation = (): JSX.Element => {
   const [isModalOpen, setIsModalOpen] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
 
+  const ACCUEIL: Tab = {
+    label: 'Accueil',
+    path: '/',
+    icon: <HomeIcon style={{ fill: 'currentcolor' }} width="1.4rem" />,
+  };
+  const FREE_CONTENT: Tab = {
+    label: 'Publier un contenu libre',
+    path: '/contenu-libre',
+    icon: <FreeContentIcon style={{ fill: 'currentcolor' }} width="1.4rem" />,
+  };
+  const ANTHEM_PARAM: Tab = {
+    label: "Paramétrer l'hymne",
+    path: '/parametrer-hymne',
+    icon: <AnthemIcon style={{ fill: 'currentcolor' }} width="1.4rem" />,
+  };
```

Comment:
**nit:** You can move these constants outside the component. So that they are not redefined each time the component render.